pkgs/development/python-modules/pywebview/default.nix

```diff
@@ -50,21 +50,17 @@ buildPythonPackage rec {
   ];
 
   checkPhase = ''
-    # Cannot create directory /homeless-shelter/.... Error: FILE_ERROR_ACCESS_DENIED
-    export HOME=$TMPDIR
-    # QStandardPaths: XDG_RUNTIME_DIR not set
-    export XDG_RUNTIME_DIR=$HOME/xdg-runtime-dir
+    # a Qt wrapper is required to run the Qt backend
+    # since the upstream script does not have a way to disable tests individually pytest is used directly instead
```

Comment:
It'd be great if you made upstream aware of this so that they can both fix the tests and provide a mechanism for disabling individual tests. Please link the issue if you do.

Reply:
(While you're at it you could also ask them to make these faster and/or parallelise them; my god are these *slow*.)

Reply:
I already had [the upstream issue](https://github.com/r0x0r/pywebview/issues/1525) opened to figure out why the *run.sh* is structured the way it is, and I only just got a response that I'll have to ponder for a bit.
Considering that the tests most likely do actual graphical things (otherwise the *xvfb-run* wouldn't be needed) I wouldn't get my hopes up about parallel execution.

It is on my todo list.